website/docs/core-concepts/writing-code/workflows.md

```diff
@@ -2,155 +2,161 @@
 
-* Click on the **+** icon next to the **queries/js** and create a new query.
-* Select `DatabaseName` for which you want to add Query.
-* Rename the query to `fetchUsersList.`
+<VideoEmbed host="youtube" videoId="rlk5c0HWW5o" title="Write Custom Code using JS label" caption="Write Custom Code using JS label"/>
 
-Assuming that the table name is Users. Add the following code to the query editor.
+You can also configure actions using the GUI and then click the JS icon next to the event/property to view the JavaScript equivalent of the configuration. This can serve as a starting point and help you configure workflows according to your specific needs. This gives you more flexibility and options to configure complex workflows that can't be handled by the GUI alone.
 
-:::info
-Replace **users** with the **table name** in your database.
-:::
+### Conditional execution
+Write custom JavaScript code enables you to handle conditional execution of operations. For example, you wish to execute different queries based on user selection, and bind the data to a table widget. You can do so, by enabling the JS label next to the table data property and add below code:
 
-```sql
-SELECT * FROM users ORDER BY id LIMIT 10;
+```javascript
+{{
+  Choose_operations.selectedOptionValue === 'Fetch Users from DB' ? FetchUsersFromDB.data : FetchUsersFromAPI.data.users
+}}
 ```
 
-An execution call to the query `fetchUsersList.run()` will throw an error as table data expects data and cannot perform query execution. However, you can read the query response and use it in the Table data to display records by using `{{<QUERY_NAME.data>}}(fetchUsersList.data)`. Appsmith handles the query execution **on page load**. You can modify the query settings from the settings tab if you wish not to execute the query **on page load**.
-
-## **Events**
-
-You can trigger the actions by binding them with Async fields. For example, you want to show a success message on the submission of a button by using `showAlert()`. You'll have to bind the \[`showAlert()`]\(Add a link to Actions - Show Alert method) function on the `onClick()` event.
-
-:::info
-You can only use the **global actions** provided out-of-the-box by Appsmith in the **async** **fields**, which are events or actions.
-:::
+<VideoEmbed host="youtube" videoId="N_M-i9dnweo" title="Trigger multiple operations based on user input" caption="Trigger multiple operations based on user input"/>
 
- <VideoEmbed host="youtube" videoId="tjJIDkoCyQE" title="Global Functions" caption="Global Functions"/>  
+In this scenario, `Choose_operations` is a select widget that allows the user to select various options. Based on the selected option, the corresponding query is triggered and the generated data is displayed in a table widget.
 
-### Handling Success / Error
+You can also chain the queries to execute conditionally based on the value of a widget or the response of a Query.
```

Comment:
This is the same as above right?

Reply:
This is for Async fields. Added events to the content.